packages/cli/cli/src/cli.ts

```diff
@@ -63,7 +63,8 @@ async function runCli() {
     } catch (error) {
         if (error instanceof FernCliError) {
             if (cliContext.getLogLevel() === LogLevel.Debug) {
-                cliContext.logger.error(`${error.name}: ${error.message}\n${error.stack}`);
+                cliContext.logger.error(`${error.name}: ${error.message}`);
+                cliContext.logger.error(`${error.stack}`);
```

Comment:
Not sure if this was an intentional change but the reason for the `\n` was so the prefix only appears one:
```
[fern-typescript]: error message
                   at bundle.js:3213 ...
```

vs

```
[fern-typescript]: error message
[fern-typescript]: at bundle.js:3213 ...
```

Reply:
gotcha i made that change it just looked weird

Reply:
<img width="1427" alt="Screen Shot 2022-12-02 at 5 45 04 PM" src="https://user-images.githubusercontent.com/10870189/205416599-43773079-2ea3-4c76-851d-d3b6fba3ff4a.png">

Reply:
Lmao looks like we can just scrap the first log then and just log the stack

Reply:
Oh got it there’s no message

Reply:
Cool yeah what you have looks perfect

Reply:
And maybe we only log the first line if there’s a non empty message?

Reply:
perfect will change that